services/supervisor/src/transactions.rs

```diff
@@ -89,6 +120,126 @@ impl ValidateInput for StartService {
 }
 
 impl SupervisorInterface for Supervisor {
+    fn propose_config_change(
+        &self,
+        context: TransactionContext,
+        propose: ConfigPropose,
+    ) -> Result<(), ExecutionError> {
+        let ((_, author), fork) = context
+            .verify_caller(Caller::as_transaction)
+            .ok_or(DispatcherError::UnauthorizedCaller)?;
+        let schema = Schema::new(context.instance.name, fork);
+
+        // Verifies that transaction author is validator.
```

Comment:
Here if `verified` but bellow is `check`.

Reply:
Changed `check` to `verified`